Lib/asyncio/tools.py

```diff
@@ -232,20 +234,56 @@ def _get_awaited_by_tasks(pid: int) -> list:
         sys.exit(1)
 
 
-def display_awaited_by_tasks_table(pid: int) -> None:
+class TaskTableOutputFormat(Enum):
```

Comment:
Any reason not to use a `StrEnum` combined with `enum.auto()`?

Reply:
changed 👍